app/utils/DateTimeUtils.scala

```diff
@@ -30,6 +29,7 @@ object DateTimeUtils {
   val rfc1123DateTimeFormatter: DateTimeFormatter =
     DateTimeFormatter.ofPattern(rfc1123DateTimePattern).withZone(ZoneId.systemDefault())
 
-  def dateTimeWritesIsoUtc: Writes[DateTime] = (d: org.joda.time.DateTime) =>
-    JsString(d.toString(ISODateTimeFormat.dateTimeNoMillis().withZoneUTC()))
+  def dateTimeWritesIsoUtc: Writes[LocalDateTime] = (d: java.time.LocalDateTime) =>
+    JsString(d.atOffset(ZoneOffset.UTC).truncatedTo(
```

Comment:
Could you please update the time format as below as we don't need Mili seconds while making the API call?

d.atOffset(ZoneOffset.UTC).truncatedTo(ChronoUnit.SECONDS).format(DateTimeFormatter.ISO_DATE_TIME)